client/signup/config/flows.js

```diff
@@ -55,7 +55,7 @@ function getPostsDestination( dependencies ) {
 function getRedirectDestination( dependencies ) {
 	if (
 		dependencies.oauth2_redirect &&
-		dependencies.oauth2_redirect.match( /^https%3A%2F%2Fpublic-api.wordpress.com/i )
+		startsWith( dependencies.oauth2_redirect, 'https://public-api.wordpress.com' )
```

Comment:
Cool. If you're going to use lodash's startsWith, you don't need the existence check on the line above. `String` also has a native `startsWith` method that we polyfill.

Reply:
Ah ok cool, I wasn't sure if we had the polyfill...

Reply:
Using the native function now, for performance :)

Reply:
Looks great!